Libs/Widgets/ctkErrorLogWidget.cpp

```diff
@@ -186,6 +191,32 @@ void ctkErrorLogWidget::setColumnHidden(int columnId, bool hidden) const
   d->ErrorLogTableView->setColumnHidden(columnId, hidden);
 }
 
+// --------------------------------------------------------------------------
+void ctkErrorLogWidget::setLayoutOrientation(Qt::Orientation orientation)
+{
+  Q_D(ctkErrorLogWidget);
+  if (d->LayoutOrientation == orientation)
+    {
+    return;
+    }
+
+  d->ctkErrorLogGridLayout->removeWidget(d->ErrorLogDescription);
+  int fromRow = (orientation == Qt::Orientation::Vertical ? 2 : 0);
```

Comment:
I find this code hard to read. Could you please reorganize it to something more explicit like this?
(if you really want to be fancy then you could get `errorLogTableViewRowIndex` from the layout instead of hardcoding)

```
if (orientation==Qt::Orientation::Vertical)
{
  // Description is below message table
  const int errorLogTableViewRowIndex = 1;
  d->ctkErrorLogGridLayout->addWidget(d->ErrorLogDescription, errorLogTableViewRowIndex + 1, 0, 1, 1);
}
else
{
  // Description is in a second column, beside the message table.
  // Specifying rowSpan = -1 ensures the widget fills the entire second column.
  d->ctkErrorLogGridLayout->addWidget(d->ErrorLogDescription, 0, 1, -1, 1);
}
```